rllib/utils/annotations.py

```diff
@@ -118,6 +118,7 @@ def OverrideToImplementCustomLogic(obj):
         ...     # ... w/o calling the corresponding `super().loss()` method.
         ...     ... # doctest: +SKIP
     """
+    obj.__is_overriden__ = False
```

Comment:
Actually, I forgot, we had these decorators :) We should probably use these very consequently in a separate PR. Also for the Trainer sub-classes.

Reply:
yeah, we seem to have everything ...